### PR TITLE
Bump version and housekeeping

### PR DESCRIPTION
**Pull request by tubedude**
Do you care to comment? @milus, @rootedbox

### CODE REVIEW

File: lib/xirr/base.rb [hunk 1]
```diff
@@ -24,16 +25,9 @@ def periods_from_start(date)
     # @return [BigDecimal]
     def xnpv(rate)
       cf.inject(0) do |sum, t|
-        sum + (xnpv_c rate, t.amount, periods_from_start(t.date))
+        # sum + (xnpv_c rate, t.amount, periods_from_start(t.date))
```
Comment (milus) [inline lib/xirr/base.rb:28]: Development leftover? Could be removed.

File: lib/xirr/newton_method.rb [hunk 2]
```diff
@@ -1,11 +1,12 @@
 # frozen_string_literal: true
 require 'bigdecimal/newton'
-include Newton
 
 module Xirr
   # Class to calculate IRR using Newton Method
   class NewtonMethod
     include Base
+    include Newton
```
Comment (milus) [inline lib/xirr/newton_method.rb:8]: 👍

**Review by milus — APPROVED** [review]
lgtm

**Comment by rootedbox** [comment]
LGTM

**Pull request merged**